04/File.js

```diff
@@ -1,8 +1,43 @@
 import React from 'react';
+import { v4 as uuid } from 'uuid'
 
 class File extends React.Component {
+    handleFilesSubmit = event => {
+        event.preventDefault();
+        const files = Array.from(event.target.files);
+        files.map(file => {
+            if (file.type === 'text/plain') {
+                const newObj = this.readFile(file);
+            }
+        })
+    }
+    readFile = (file) => {
+        const { getFile } = this.props;
+        const reader = new FileReader();
+        reader.readAsText(file);
+        reader.onload = () => {
+            const innerText = reader.result;
+            const newObj = this.createObject(file, innerText);
+            this.props.getFile(newObj)
```

Comment:
Ponieważ tutaj wywołujesz tą funkcję to kontekst jest inny.
Dlatego musisz użyć inicjalizatora właściwości lub powiązać kontekst dla `this` przy pomicy `.bind()`.